src/components/table/components/table-row/table-row.js

```diff
@@ -58,18 +74,18 @@ export const TableRow = ({
       hasStickyHeader={hasStickyHeader}
       {...row.getRowProps({ style })}
     >
-      {row.cells.map(cell => {
-        const { key, ...cellProps } = cell.getCellProps()
-        return (
-          <TableCell
-            key={key}
-            cell={cell}
-            selectedRowIds={selectedRowIds}
-            {...cellProps}
-            customProps={customProps}
-          />
-        )
-      })}
+      {Object.keys(groupedRows).length ? (
```

Comment:
You can have this in a separate function , in order to be more clear whats going on here. Like the `renderCells` func